dubbo-configcenter-extensions/dubbo-configcenter-consul/src/test/java/org/apache/dubbo/configcenter/consul/ConsulDynamicConfigurationTest.java

```diff
@@ -48,7 +50,7 @@ public static void setUp() throws Exception {
         configCenterUrl = URL.valueOf("consul://127.0.0.1:" + consul.getHttpPort());
 
         configuration = new ConsulDynamicConfiguration(configCenterUrl);
-        client = Consul.builder().withHostAndPort(HostAndPort.fromParts("127.0.0.1", consul.getHttpPort())).build();
+        client = Consul.builder().withHostAndPort(HostAndPort.fromParts("127.0.0.1", consul.getHttpPort())).withReadTimeoutMillis(TimeUnit.SECONDS.toMillis(11)).build();
```

Comment:
why it is 11

Reply:
Because consul will check `cacheWatchSeconds` (default 10s) and `networkReadMillis` size, if `networkReadMillis` is less than `cacheWatchSeconds`, it will give an error and fail the case

![image](https://github.com/apache/dubbo-spi-extensions/assets/56506697/1bae4e52-7bf3-469c-9e31-7cf9553abeb2)

Reply:
![Uploading image.png…]()